src/App.js

```diff
@@ -90,6 +91,10 @@ class App extends Component {
     }
     this.identifyFlow();
     AppState.addEventListener('change', this.handleAppStateChange);
+
+    // Wallet connect v2
+    walletConnectInit(store);
```

Comment:
I guess that instead of passing it as a param you could import it directly on that file